app/javascript/lib/phoenix-graphql-backend.js

```diff
@@ -0,0 +1,46 @@
+import { Socket as PhoenixSocket } from 'phoenix';
+import { setContext } from 'apollo-link-context';
+import { hasSubscription } from '@jumpn/utils-graphql';
+import { HttpLink } from 'apollo-link-http';
+import { split } from 'apollo-link';
+import Cookies from 'js-cookie';
+import * as AbsintheSocket from '@absinthe/socket';
+import { createAbsintheSocketLink } from '@absinthe/socket-apollo-link';
+
+const webSocketURL = new URL(process.env.BACKEND_URL);
+
+webSocketURL.protocol = 'ws:';
```

Comment:
:100: nice way to switch the protocol without find/replace

Reply:
Thanks!